cargo-shuttle/src/lib.rs

```diff
@@ -101,13 +101,16 @@ impl Shuttle {
         args: ShuttleArgs,
         provided_path_to_init: bool,
     ) -> Result<CommandOutcome> {
-        trace!("running local client");
-
-        if args.api_url.as_ref().is_some_and(|s| s.ends_with('/')) {
-            println!(
-                "WARNING: API URL is probably incorrect. Ends with '/': {}",
-                args.api_url.clone().unwrap()
-            );
+        if let Some(ref s) = args.api_url {
+            if s != API_URL_DEFAULT {
+                eprintln!("INFO: Targetting non-standard API: {}", s);
+            }
+            if s.ends_with('/') {
+                eprintln!(
+                    "WARNING: API URL is probably incorrect. Ends with '/': {}",
+                    args.api_url.clone().unwrap()
+                );
+            }
```

Comment:
nit: the expectation is to not have a trailing slash so it's probably better to remove it if there and avoid printing this warning altogether. WDYT?

Reply:
Don't want to obfuscate the parsing logic too much. This way is transparent in the sense that we are concatenating strings instead of "URL building".